egs/swbd/s5c/local/score_basic.sh

```diff
@@ -45,7 +46,7 @@ function filter_text {
   perl -e 'foreach $w (@ARGV) { $bad{$w} = 1; }
    while(<STDIN>) { @A  = split(" ", $_); $id = shift @A; print "$id ";
      foreach $a (@A) { if (!defined $bad{$a}) { print "$a "; }} print "\n"; }' \
-   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION'
+   '[NOISE]' '[LAUGHTER]' '[VOCALIZED-NOISE]' '<UNK>' '%HESITATION' '[noise]' '[laughter]' '[vocalized-noise]' '<unk>' '%hesitation'
```

Comment:
You seem to have discovered a problem in the scoring script.  How much WER difference does this make?
We should fix it but we have to be careful about results changing (only affects train_dev).
No need to add uppercase+lowercase-- it's all lowercase.